.env.example

```
@@ -0,0 +1,73 @@
+# Default: /usr/bin/google-chrome
+CHROME_PATH=
+
+# AWS Configuration
+# This module makes use of AWS SDKs for S3
+# The SDK automatically gets credentials from EKS, credentials store or env variables (need to follow AWS standardised variable names)
+# The order is defined here - https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/setting-credentials-node.html
+# Depending on your environment and configuration, you may want to configure the below (and additional) AWS properties
+# Or omit them if using temporary variables exposed through EKS/IAM roles (recommended for production)
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+AWS_REGION=fr-par
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AWS_REGION key should go before the AWS_SECRET_ACCESS_KEY key

(UnorderedKey)